cli/search/crawler.go

```diff
@@ -65,8 +67,18 @@ func collectBundleReferences(searchCtx *SearchCtx, baseUrl string,
 		r.Headers.Add("Authorization", "Basic "+authEncoded)
 	})
 
+	c.OnError(func(response *colly.Response, error error) {
+		err = error
+	})
+
 	c.Visit(eeUrl.String())
 	c.Wait()
+	if err != nil {
+		warnMsg := "Don't use your credentials from personal account in tarantool.io. " +
```

Comment:
How do we know this is an auth error? Don't we need to check a response code for...I'm not sure, 401?

Reply:
Maybe:
"tarantool.io login credentials cannot be used for tarantool-ee installation. Contact your system administrator for the right credentials."
?

Reply:
Fixed, thank you.